docs/book/src/quinn/data-transfer.md

```diff
@@ -0,0 +1,172 @@
+# Data Transfer
+
+In the [previous chapter](set-up-connection.md) we characterized how to set up an [Endpoint][Endpoint]
+and then get access to a [NewConnection][NewConnection] instance.
+Now we will continue with the subject of sending data over this connection.
+
+## Multiplexing
+
+A QUIC stream can be compared to a TCP stream, eventhough if it is based on UDP. 
+The big difference is that you are not limited to a single stream. 
+You can open multiple streams between two peers, this is also called multiplexing.
```

Comment:
This doesn't seem accurate. "multiplexing" is the act of combining data from multiple streams in a single stream, which isn't really what you're referring to here.

Reply:
Agree, I will rephrase as you posted.